src/Tools/Microsoft.VisualStudio.Templates.Editorconfig.Wizard/Microsoft.VisualStudio.Templates.Editorconfig.Wizard.csproj

```diff
@@ -11,11 +11,13 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" />
     <PackageReference Include="Microsoft.VisualStudio.LanguageServices" />
-    <PackageReference Include="Microsoft.VisualStudio.SDK" />
-    <PackageReference Include="Microsoft.VisualStudio.Telemetry" />
-    <PackageReference Include="Microsoft.VisualStudio.TemplateWizardInterface" />
-    <PackageReference Include="StreamJsonRpc" />
+    <PackageReference Include="Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.9.35087-preview.1" />
```

Comment:
It's a bit odd we're mixing central package versions and inline versions here. Did we want to standardize to one?

Reply:
I moved over to standardize. The system had some issues with the RPC package versioning so I'm going to leave those inline.